components/UserForm.js

```diff
@@ -0,0 +1,105 @@
+export default function UserForm({ client, handleSubmit, handleInputChange }) {
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="firstName">Name{client.firstName}</label>
+      <input
+        type="text"
+        id="firstName"
+        name="firstName"
+        //defaultValue={curUser.firstName}
+        value={client.firstName}
+        onChange={handleInputChange}
+        required
+      />
+
+      <label htmlFor="lastName">LastName</label>
+      <input
+        type="text"
+        id="lastName"
+        name="lastName"
+        //defaultValue={curUser.lastName}
+        value={client.lastName}
+        onChange={handleInputChange}
+        required
+      />
+
+      <label htmlFor="email">email</label>
+      <input
+        type="email"
+        id="email"
+        name="email"
+        //defaultValue={curUser.email}
+        value={client.email}
+        onChange={handleInputChange}
+        required
+      />
+
+      <label htmlFor="phoneNumber">PhoneNumber</label>
+      <input
+        type="tel"
+        id="phoneNumber"
+        name="phoneNumber"
+        //defaultValue={curUser.phoneNumber}
+        value={client.phoneNumber}
+        onChange={handleInputChange}
+        required
+      />
+
+      <label htmlFor="zib">ZIB</label>
+      <input
+        type="text"
+        id="zib"
+        name="zib"
+        //defaultValue={curUser.zib}
+        value={client.zib}
+        onChange={handleInputChange}
+        required
+      />
```

Comment:
Do you mean ZIP for "Postleitzahl"? Then it is with a P :) Otherwise, forget this comment!

Reply:
typo☹️